apps/client/src/app/modules/pricing/pricing/pricing.component.ts

```diff
@@ -226,12 +226,12 @@ export class PricingComponent implements AfterViewInit {
         const amount = this.pricingService.getAmount(list.$key, row, false);
         let priceString: string;
         if (price.hq > 0 && amount.hq > 0) {
-          priceString = `${price.hq.toLocaleString()}gil x${amount.hq}(HQ) (${price.hqServer})`;
+          priceString = `${price.hq.toLocaleString()}gil x${amount.hq}(HQ) (${this.getWorldName(price.hqServer)})`;
```

Comment:
Why not use the pipe instead?

Reply:
Referencing to codes like `this.i18n.getName(this.l12n.getItem(row.id))`, I think it's reasonable (not perfect, sure). Also calling pipes will result in 2 more dependency injections.